source/diagrams/manage-diagrams/rename-diagrams.txt

```diff
@@ -10,4 +10,47 @@ Rename Diagrams
    :depth: 1
    :class: singlecol
 
-Rename diagrams.
\ No newline at end of file
+You can rename diagrams on the Mapping tab of Relational Migrator.
+Relational Migrator displays diagram names on the ribbon tab. 
+
+About this Task
+---------------
+
+You can rename diagrams to customize your diagram properties. Relational
+Migrator shows your diagram names on the ribbon tab of the Mapping view and in
+the Diagram Properties pane. 
+
+Steps
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Navigate to the diagram you want to rename. 
+
+      Click the name of the diagram in the ribbon tab to display the
```

Comment:
As a follow-up to the comment above regarding the ribbon tab, would it help users to direct them where to look on the screen for the tab or is that something that's generally obvious?

Reply:
Just added an image!